src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -1121,4 +1121,18 @@ export function clinicalDataCountComparator(a: ClinicalDataCount, b: ClinicalDat
     else {
         return b.count - a.count;
     }
+}
+
+export function submitToPage(url:string, params: { [id: string]: string }, target?: string) {
+    window.localStorage.removeItem("legacyStudySubmission");
+    try {
+        window.localStorage.setItem("legacyStudySubmission", JSON.stringify(params))
+    } catch (e) {
+        // in case of this QuotaExceededError clear case_ids filed as this is the only one which could be longest 
+        console.log(e);
```

Comment:
hmmm. i think you need to clear localStorage, not clear the params you are trying to set.